Packs/TrendMicroVisionOne/Integrations/TrendMicroVisionOne/TrendMicroVisionOne_test.py

```diff
@@ -634,6 +636,66 @@ def test_submit_file_to_sandbox(mocker):
     assert result.outputs["code"] == "Success"
 
 
+# Mock response for submit file entry to sandbox
+def mock_submit_file_entry_to_sandbox_response(*args, **kwargs):
```

Comment:
Where is this mock used? I don't see any other reference to it in the code.

Reply:
Hey @omerKarkKatz I was initially going to use this to test against the submit file entry to sandbox but it's using the same call for submit file to sandbox so it is redundant and shouldn't be in code. That's an oversight on my part. That whole mock submit file entry to sandbox can be removed.